packages/site-demo/content/product/components/language-switcher/index.mdx

```diff
@@ -0,0 +1,25 @@
+# Language switcher
+
+**Language switchers display language options in a list and present the users choice. Use it when users need to select a language.**
```

Comment:
```suggestion
**Language switchers display language options in a list and present the option the user chose. Use it when users need to select a language.**
```
It may seem a little heavy but it disambiguates.